### ISSUE TITLE
Create wordlist folder in ~/.config/Code/User cause the extension stop working

### ISSUE THREAD

**Issue by gky99** [yzhang-gh/vscode-dic-completion#40]
I notice that if I add the folder ~/.config/Code/User/wordlist, the completion will stop working. Delete the folder and restart vscode will fix it.

Environment is Manjaro 21, KDE

**Comment by yzhang-gh** [comment]
Thanks for the feedback.

Did you see any error messages from the console? (menu bar>help>toggle developer tools)

**Comment by gky99** [comment]
```
Activating extension 'yzhang.dictionary-completion' failed: EISDIR: illegal operation on a directory, read.
$onExtensionActivationError	@	mainThreadExtensionService.ts:111
```

**Comment by yzhang-gh** [comment]
Probably it is because of the `~`. You can replace it with an absolute path.

**Comment by gky99** [comment]
No, I mean the creation of that folder, not adding it to the external dict list.

**Comment by yzhang-gh** [comment]
I see. It is because this extension assumes `~/.config/Code/User/wordlist` to be a file but not a folder. You can use command `openUserDict` to create this file.
